### PR TITLE
Refactored graph viewer page

### LINKED ISSUE #1: Refactor client-side code
Mostly in App.vue

### PR DESCRIPTION
**Pull request by tch1001**
Removed all previous code from neo4j client and replace with basic graph explorer from v-network-graph

### CODE REVIEW

File: src/graphData.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    //   // console.log('i')
+    // } else if (type == 'edge:pointerover') {
+    //   const obj = edges[event.edge]
+    //   updateDataPanel(obj.objInfo, obj)
+    // } else if (type == 'edge:select') {
+    //   document.edgeSelect(event)
+    //   // console.log(type, event)
+    // }
+  },
+}
```
Comment (bushshrub) [inline src/graphData.ts]: Delete these

**Review by bushshrub — CHANGES_REQUESTED** [review]
Some cleanup needed

File: src/App.vue [hunk 2]
```diff
@@ -15,26 +15,21 @@
       <section class="modal-card-body">
         <form>
           <div class="row">
-            <label class="label" for="server-url">Server URL</label>
+            <label class="label" for="server-url">Username/Email:</label>
```
Comment (bushshrub) [inline src/App.vue]: We don't need the colon here

File: src/App.vue [hunk 3]
```diff
@@ -15,26 +15,21 @@
...
             <div class="control">
-              <input class="input" type="text" id="server-url" value="neo4j+s://ae6e3bf9.databases.neo4j.io" />
-            </div>
-          </div>
-
-          <div class="row">
-            <label class="label" for="server-username">Username</label>
-            <div class="control">
-              <input class="input" type="text" id="server-username" value="neo4j"/>
+              <input class="input" type="text" id="server-url" value="" />
```
Comment (bushshrub) [inline src/App.vue]: Also, this id should not be `#server-url` anymore

File: src/App.vue [hunk 4]
```diff
@@ -15,26 +15,21 @@
...
             </div>
           </div>
 
           <div class="row">
             <label class="label" for="server-password">Password</label>
             <div class="control">
-              <input class="input" type="password" id="server-password" value="pxHM45j7jD54_tvddMklXySWjbafvZfv01-B_GIDpVU"/>
+              <input class="input" type="password" id="server-password" value=""/>
```
Comment (bushshrub) [inline src/App.vue]: Ditto.

File: src/App.vue [hunk 5]
```diff
@@ -15,26 +15,21 @@
...
             </div>
           </div>
 
+            <!-- TODO login with google -->
```
Comment (bushshrub) [inline src/App.vue]: Create a task in the project board for this instead

File: src/App.vue [hunk 6]
```diff
@@ -168,605 +132,31 @@
...
+const dataPanel = {};
+const graph = graphData.graph
+const nodes = graphData.nodes
+const edges = graphData.edges
+const configs = graphData.configs
+const layouts = graphData.layouts
+const selectedNodes = graphData.selectedNodes
+const selectedEdges = graphData.selectedEdges
+const d3ForceEnabled = graphData.d3ForceEnabled
+const eventHandlers = graphData.eventHandlers
```
Comment (bushshrub) [inline src/App.vue]: Is this a requirement by `v-network-graph`? Otherwise, it would be cleaner to shove them into a dictionary

**Review by bushshrub — CHANGES_REQUESTED** [review]

**Comment by tch1001** [comment]
Ready for review @xiurobert

File: src/graphData.ts [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+    //   const obj = nodes[event.node]
+    //   updateDataPanel(obj.objInfo, obj)
+    // } else if (type == 'node:pointerout') {
+    //   // console.log('i')
+    // } else if (type == 'edge:pointerover') {
+    //   const obj = edges[event.edge]
+    //   updateDataPanel(obj.objInfo, obj)
+    // } else if (type == 'edge:select') {
+    //   document.edgeSelect(event)
+    //   // console.log(type, event)
```
Comment (bushshrub) [inline src/graphData.ts]: Can we delete these

**Review by bushshrub — CHANGES_REQUESTED** [review]

**Review by bushshrub — APPROVED** [review]

**Comment by bushshrub** [comment]
You should re open this PR @tch1001

**Pull request merged**